documentation/airport-guides/krdu.md

```diff
@@ -0,0 +1,47 @@
+### About this facility
+Raleigh-Durham Combined TRACON and ATCT (RDU), Morrisville, North Carolina, USA
+257,412 annual operations in 2017 ([FAA ATADS](https://aspm.faa.gov/opsnet/sys/Tracon.asp))
+
+This level 9 combined TRACON/ATCT facility controls a number of airfields in Greater Raleigh. Since this is a combined facility, the TRACON and ATC Tower are co-located at Raleigh-Durham International Airport (KRDU).
+
+### Airspace
+The RDU TRACON airspace encompasses a large area around Great Raleigh extending from the surface up to 12,000 feet. Class C airspace exists with tops of 4,400 feet over the Raleigh and Durham areas.
+
+### Instrument Procedures
+Although departing aircraft must be vectored to their initial fixes, the STAR's are very automated, taking a lot off weight off the controllers. Arrivals must still be vectored to their final approach course.
```

Comment:
the STARs are very automated (without the apostrophe)
taking a lot of weight off (extra f)